Resources/Prototypes/Entities/Objects/Weapons/Melee/white_cane.yml

```diff
@@ -0,0 +1,26 @@
+- type: entity
+  parent: BaseItem
+  id: WhiteCane
+  name: white cane
+  description: This isn't for you. It's for the people who can't figure out you're blind when you ask if cargo is the bar.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/white_cane.rsi
+    state: icon
+  - type: Item
+    size: 15
+    sprite: Objects/Weapons/Melee/white_cane.rsi
+  - type: MeleeWeapon
+    bluntStaminaDamageFactor: 1.60 
+    range: 1.85
```

Comment:
WHy the overwrites?

Reply:
No particular reason. I have no real coding experience and didn't realize that this was unnecessary. I removed the lines